INSTALL.markdown

```diff
@@ -127,15 +127,15 @@ DXX-Rebirth can be built from the Terminal (via SCons) without Xcode; to build u
 
 ##### [Homebrew](https://github.com/Homebrew/homebrew/)
 * **brew install
- gcc5
  scons
  sdl
  sdl\_image
  sdl\_mixer
+ libpng
  physfs**
 
 ### Building
-Once prerequisites are installed, run **scons** *options* to build.  By default, both D1X-Rebirth and D2X-Rebirth are built.  To build only D1X-Rebirth, run **scons d1x=1**.  To build only D2X-Rebirth, run **scons d2x=1**.
+Once prerequisites are installed, run **scons** *options* to build.  By default, both D1X-Rebirth and D2X-Rebirth are built.  To build only D1X-Rebirth, run **scons d1x=1**.  To build only D2X-Rebirth, run **scons d2x=1**.  If using macOS without framework builds of the SDL libraries (such as with Homebrew), run **scons mac_libs=1**.
```

Comment:
The phrasing here looks awkward to me.  The rest of the paragraph is very general instruction, and then one platform-specific parameter is advertised here.  None of the other user settings are advertised in this file, whether or not platform-specific.  I think it would be better to have the `--help` output mention this option, then add some general text here telling the user to see `--help` for a list of options, and possibly add some fallback handling in the SConf tests so that on failure to find the framework, the test suggests the option to ignore frameworks.  I don't know if that last point is easy though.

Reply:
That's fair.  I wasn't sure where a good place would be to note it, but I guess the help output would make sense.

Regarding fallback handling, there are definitely ways it could be more robust.  In my ideal world, the tests would test for an equivalent library if it failed to find a framework and use that instead if it finds it, but that was more complicated a task than I really wanted to tackle right now.

Reply:
That makes sense.  I can't do the fallback work for you since I have no way to test it.  However, I hope having the flag that you proposed will make Homebrew easy enough to use that the fallback is not needed.